tf-modules/archive/sf_event_sqs_to_db_records.tf

```diff
@@ -60,6 +60,16 @@ data "aws_iam_policy_document" "sf_event_sqs_to_db_records_lambda" {
     resources = [for b in flatten([var.public_buckets, var.protected_buckets, var.private_buckets, var.system_bucket]) : "arn:aws:s3:::${b}/*"]
   }
 
+  statement {
```

Comment:
This set of changes adds the DLQ target to the env vars, and updates the lambda to be able to send messages to the DLQ in case of a message write failure